v1/conftest.py

```diff
@@ -61,6 +61,34 @@ def block_data(account_data, account, encoded_account_number, random_encoded_acc
     )
 
 
+@pytest.fixture
+def block_data_unique_recipients(
+    account_data, account, encoded_account_number, random_encoded_account_number, self_configuration
+):
+    signing_key, account_number = account_data
+    primary_validator = self_configuration.primary_validator
+
+    yield generate_block(
+        account_number=account_number,
+        balance_lock=encoded_account_number,
+        signing_key=signing_key,
+        transactions=[
+            {
+                'amount': self_configuration.default_transaction_fee,
+                'recipient': self_configuration.account_number
+            },
+            {
+                'amount': primary_validator.default_transaction_fee,
+                'recipient': primary_validator.account_number
+            },
+            {
+                'amount': Faker().pyint(),
```

Comment:
hey, here is another problem that you've spotted somewhere else :)

Reply:
:+1: Thanks. Updated.